testing/src/cypress.config.ts

```diff
@@ -34,5 +30,8 @@ export default defineConfig({
     token_url:
```

Comment:
Are these env variables in the correct place? 
If this is not the focus of the PR we can handle this later, but I believe that we are missing the proper env file or equivalent for cypress (https://docs.cypress.io/api/cypress-api/env).

Reply:
I believe so - its the right place. But yeah we can discuss if we need to move it to .env file altogether.

Reply:
I ll change them to a dev env file in a separate PR